os/tools/mkbinheader.py

```diff
@@ -134,16 +146,37 @@
         print "Error : This binary priority is not valid"
         sys.exit(1)
 
+    textsize = 0
+    rosize = 0
+    datasize = 0
+    bsssize = 0
     ram_fp = open(STATIC_RAM_ESTIMATION, 'rb')
-    # First line is not used for calculating RAM size. So throw away it.
-    ram_fp.readline()
-    ram_array = ram_fp.readline()
+    line = ram_fp.readline()
+    while line:
+        words = line.split('.')
```

Comment:
With BIN type binary, we don't need to calculate the aligned size,because that binary will not be loaded.

Reply:
Fixed